ideascube/configuration/tests/test_command_config.py

```diff
@@ -0,0 +1,340 @@
+    assert err.strip() == ''
+
+
+@pytest.mark.usefixtures('db')
+def test_get_default_config(capsys, monkeypatch):
+    monkeypatch.setattr(
+        'ideascube.configuration.registry.REGISTRY',
+        {'namespace1': {'key1': {'type': int, 'default': 0}}})
+
+    call_command('config', 'get', 'namespace1', 'key1')
+
+    out, err = capsys.readouterr()
+    assert out.strip() == '0'
+    assert err.strip() == ''
+
+
+def test_get_config_no_namespace(monkeypatch):
+    monkeypatch.setattr('ideascube.configuration.registry.REGISTRY', {})
+
+    with pytest.raises(CommandError) as exc_info:
+        call_command('config', 'get', 'namespace1', 'key1')
+
+    assert (
+        'Unknown configuration namespace: "namespace1"') in str(exc_info.value)
+
+
+def test_get_no_config(monkeypatch):
+    monkeypatch.setattr(
+        'ideascube.configuration.registry.REGISTRY', {'namespace1': {}})
+
+    with pytest.raises(CommandError) as exc_info:
+        call_command('config', 'get', 'namespace1', 'key1')
+
+    assert (
+        'Unknown configuration key: "namespace1.key1"') in str(exc_info.value)
+
+
+@pytest.mark.usefixtures('db')
+def test_get_invalid_config(capsys, monkeypatch, user):
+    monkeypatch.setattr(
+        'ideascube.configuration.registry.REGISTRY',
+        {'namespace1': {'key1': {'type': int, 'default': 0}}})
+
+    # Make an invalid config, this should never happen in theory
+    Configuration(
+        namespace='namespace1', key='key1', value='foo', actor=user).save()
+
+    call_command('config', 'get', 'namespace1', 'key1')
+
+    out, err = capsys.readouterr()
+    assert out.strip() == '0'
+    assert err.strip().split(':') == [
+        'ERROR', 'ideascube.configuration',
+        "The stored value for namespace1.key1='foo' is of type <class 'str'> "
+        "instead of <class 'int'>. This should never have happened."]
+
+
+def test_help_for_config(capsys, monkeypatch):
+    monkeypatch.setattr(
+        'ideascube.configuration.registry.REGISTRY', {
+            'namespace1': {
+                'key1': {
+                    'summary': 'A configuration key',
+                    'pretty_type': 'A string',
+                    'default': 'the default',
+                },
+            },})
+
+    call_command('config', 'help', 'namespace1', 'key1')
+
+    out, err = capsys.readouterr()
+    assert out.strip().split('\n') == [
+        '# namespace1 key1',
+        '',
+        'A configuration key',
+        '',
+        '* type: A string',
+        "* default value: 'the default'"]
+    assert err.strip() == ''
+
+
+def test_help_for_config_no_namespace(monkeypatch):
+    monkeypatch.setattr('ideascube.configuration.registry.REGISTRY', {})
+
+    with pytest.raises(CommandError) as exc_info:
+        call_command('config', 'help', 'namespace1', 'key1')
+
+    assert (
+        'Unknown configuration namespace: "namespace1"') in str(exc_info.value)
+
+
+def test_help_for_no_config(monkeypatch):
```

Comment:
I renamed it to `test_help_for_config_no_key to be more consistent with the rest of the tests in the latest push to this branch.